app/routers/invitation.py

```diff
@@ -1,65 +0,0 @@
-from typing import List, Union
```

Comment:
Why did you remove it?

Reply:
I didn't remove it, I converted it. Now the `notification` route contains `messages` and `invitations`